mechanism/scram/src/main/java/org/wildfly/security/mechanism/scram/ScramServer.java

```diff
@@ -240,6 +262,16 @@ public ScramInitialServerResult evaluateInitialResponse(final ScramInitialClient
         return new ScramInitialServerResult(new ScramInitialServerMessage(clientMessage, serverNonce, salt, iterationCount, messageBytes), password);
     }
 
+    /**
+     * Parses the final client message for the {@link ScramFinalClientMessage} from parsed information.
+     * Also checks if the message have all necessary properties.
```

Comment:
Just total minors,

s/Parses the final client message for the {@link ScramFinalClientMessage} from parsed information./Parses the final client message and constructs the {@link ScramFinalClientMessage} from this parsed information.

s/message have/message has